### PR TITLE
 [#1403] CLI Docs Revamp 

### PR DESCRIPTION
**Pull request by paulodiniz**
Fixes https://github.com/airyhq/airy/issues/1403

### CODE REVIEW

**Review by lucapette — APPROVED** [review]
:shipit:

**Review by lucapette — APPROVED** [review]

File: docs/docs/cli/introduction.md [hunk 1]
```diff
@@ -16,12 +16,17 @@ Airy Core instances directly from your terminal.
...
 Installing the Airy CLI is easy and straightforward. You can follow the next
 steps for a quick setup:
 
-- [Step 1: Check the requirements](installation.md#step-1-check-the-requirements)
-- [Step 2: Install the Airy CLI](installation.md#step-2-install-the-airy-cli)
-- [Step 3: Verify your installation](installation.md#step-3-verify-your-installation)
+- [Step 1: Check the requirements](introduction.md#step-1-check-the-requirements)
+- [Step 2: Install the Airy CLI](introduction.md#step-2-install-the-airy-cli)
+- [Step 3: Verify your installation](introduction.md#step-3-verify-your-installation)
+- [Step 4: Initialize the setup](introduction.md#step-4-initialize-the-setup)
```
Comment (paulodiniz) [inline docs/docs/cli/introduction.md:29]: Why is this being added? I rebased on top of develop

Comment (pascal-airy) [reply docs/docs/cli/introduction.md:29]: I think it was generated from the markdown plugin. Looks correct anyhow

File: docs/cli-doc/generate_cli_docs.go [hunk 2]
```diff
@@ -39,44 +39,62 @@ func main() {
...
 
+	if err := genCommandNamespaceDoc(cmd, w, level); err != nil {
+		return err
+	}
 	for _, c := range subcommands {
 		if !c.IsAvailableCommand() || c.IsAdditionalHelpTopicCommand() {
 			continue
 		}
-		if err := genMarkdownTreeCustom(c, w); err != nil {
+		if err := genMarkdownTreeCustom(c, w, level+"#"); err != nil {
```
Comment (pascal-airy) [inline docs/cli-doc/generate_cli_docs.go:64]: nice 👌

**Review by pascal-airy — APPROVED** [review]
lgtm

**Pull request merged**